Manual/VCGen.lean

```diff
@@ -55,12 +55,12 @@ open Std.Do
 
 ## Preconditions and Postconditions
 
-One style in which program specifications can be written is to provide a {deftech}_precondition_, which the program's caller is expected to ensure, and a {deftech}_postcondition_, which the program itself is expected to ensure.
-The program is correct if running it when the precondition holds always results in the postcondition holding.
+One style in which program specifications can be written is to provide a {deftech}_precondition_ $`P`, which the caller of a program `prog` is expected to ensure, and a {deftech}_postcondition_ $`Q`, which the `prog` is expected to ensure.
+The program `prog` satisfies the specification if running it when the precondition $`P` holds always results in the postcondition $`Q` holding.
```

Comment:
I think that if we're using `prog` as a mathematical metavariable, then it should be written in math font rather than in code font, because mono indicates to me that it's the name of a concrete, defined program.  Maybe `\mathit{prog}`?

Reply:
Great!

Reply:
(I'm happy to make this change if it's OK with you)